pkg/openstack/octavia.go

```diff
@@ -183,6 +183,11 @@ func ReconcileOctavia(ctx context.Context, instance *corev1beta1.OpenStackContro
 			octavia.Spec.Secret = instance.Spec.Secret
 		}
 
+		if octavia.Spec.DatabaseInstance == "" {
+			// octavia.Spec.DatabaseInstance = instance.Name // name of MariaDB we create here
+			octavia.Spec.DatabaseInstance = "openstack" // FIXME: see above
+		}
```

Comment:
Okay.  I see we indeed have this elsewhere [1].

[1] https://github.com/openstack-k8s-operators/openstack-operator/blob/4dd3ceb8af9234b651913b65f8a90484875ef684/pkg/openstack/neutron.go#L155-L157